runtime/rasdump/dmpsup.c

```diff
@@ -410,6 +410,7 @@ criuReloadXDumpAgents(J9JavaVM *vm, J9VMInitArgs *vmArgs)
 	IDATA result = configureDumpAgents(vm, vmArgs, FALSE);
 	unlockConfig();
 
+	Trc_trcengine_criu_criuReloadXDumpAgents_Exit(vm->mainThread, result);
```

Comment:
This should be paired with an "Enter" tracepoint, otherwise I think it will break indentation for `-Xtrace:iprint`

Reply:
@JasonFengJ9 ^^

Reply:
Created 
* https://github.com/eclipse-openj9/openj9/pull/17057